superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -35,6 +35,7 @@ import Icons from 'src/components/Icons';
 import findPermission, { isUserAdmin } from 'src/dashboard/util/findPermission';
 import { UserWithPermissionsAndRoles } from 'src/types/bootstrapTypes';
 import { RootState } from 'src/dashboard/types';
+import { Tag } from 'antd';
```

Comment:
Rather than importing the `Tag` component directly from antd, we should probably `import Label from 'src/components/Label';` since that'll be the highly reused and restyled/supported component in most (hopefully all) Superset instances.